cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_ionic_client_app/templates.xml

```diff
@@ -3,26 +3,26 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1">
 
   <templates>
-    <templateExtension ref="${variables.etoName#lower_case}-detail.page.ts" mergeStrategy="tsmerge"/>
-    <templateExtension ref="${variables.etoName#lower_case}-list.page.ts" mergeStrategy="tsmerge_override"/>
+    <templateExtension ref="${variables.etoName#lower_case}-detail.page.ts" mergeStrategy="tsmerge"/>
+    <templateExtension ref="${variables.etoName#lower_case}-list.page.ts" mergeStrategy="tsmerge_override"/>
     <templateExtension ref="app.component.ts" mergeStrategy="tsmerge_override"/>
     <templateExtension ref="app.module.ts" mergeStrategy="tsmerge_override"/>
-    <templateExtension ref="${variables.etoName#lower_case}-rest.service.ts" mergeStrategy="tsmerge_override"/>
+    <templateExtension ref="${variables.etoName#lower_case}-rest.service.ts" mergeStrategy="tsmerge_override"/>
     <templateExtension ref="${variables.etoName#lower_case}.ts" />
     <templateExtension ref="${variables.etoName#lower_case}-search-criteria.ts" />
     <templateExtension ref="paginated-list-to.ts" />
     <templateExtension ref="pageable.ts" />
     <templateExtension ref="sort.ts" />
-    <templateExtension ref="environment.android.ts" mergeStrategy="tsmerge_override"/>
-    <templateExtension ref="environment.prod.ts" mergeStrategy="tsmerge_override"/>
-    <templateExtension ref="environment.ts" mergeStrategy="tsmerge_override"/>
-    <templateExtension ref="app-routing.module.ts" mergeStrategy="tsmerge_override"/>
-    <templateExtension ref="${variables.etoName#lower_case}-detail.page.html" mergeStrategy="html-ng*"/>
-    <templateExtension ref="${variables.etoName#lower_case}-list.page.html" mergeStrategy="html-ng*"/>
-    <templateExtension ref="es.json" mergeStrategy="jsonmerge_override"/>
+    <templateExtension ref="environment.android.ts" mergeStrategy="tsmerge_override"/>
+    <templateExtension ref="environment.prod.ts" mergeStrategy="tsmerge_override"/>
```

Comment:
Can you check the line endings characters? Git thinks there are changes in your files, but there shouldn't